init-tas-vars.sh

```diff
@@ -1,12 +1,25 @@
-# init tuf and REKOR
+# init Tuf and REKOR
+# for local build and CI we set REKOR_HOST and TUF_MIRROR 
 
-RT=$(oc get routes -n rhtap-tas -o name | grep rekor-server)
-HOST=$(oc get -n rhtap-tas $RT -o jsonpath={.spec.host})
-export REKOR_HOST=https://$HOST
+echo "Checking for REKOR and TUF" 
+
+# get hosts, if errors occur or cannot find routes, set to none
+RT=$(oc get routes -n rhtap-tas -o name 2>  /dev/null | grep rekor-server )
+if [[ $RT == "" ]]; then 
+    export REKOR_HOST=none 
+else
+    HOST=$(oc get -n rhtap-tas $RT -o jsonpath={.spec.host})
+    export REKOR_HOST=https://$HOST
+fi 
+
+RT=$(oc get routes -n rhtap-tas -o name 2>  /dev/null| grep tuf)
+if [[ $RT == "" ]]; then 
+    export TUF_MIRROR=none 
```

Comment:
```suggestion
RT=$(oc get routes -n rhtap-tas -o name 2>  /dev/null | grep rekor-server )
if [[ $RT == "" ]]; then 
    export REKOR_HOST=''
else
    HOST=$(oc get -n rhtap-tas $RT -o jsonpath={.spec.host})
    export REKOR_HOST=https://$HOST
fi 

RT=$(oc get routes -n rhtap-tas -o name 2>  /dev/null| grep tuf)
if [[ $RT == "" ]]; then 
    export TUF_MIRROR='' 
```

The download-sbom-from-url-in-attestation.sh doesn't recognize `none` as a magic value, but empty strings should work